src/client/components/app/Transactions/Transaction.tsx

```diff
@@ -109,7 +110,7 @@ export const Transaction: FC<TransactionProps> = (props) => {
 
   if (transactionCompleted) {
     return (
-      <StyledTransaction onClose={onClose} header={transactionLabel} {...props}>
+      <StyledTransaction onClose={onClose} header={transactionLabel} {...restProps}>
```

Comment:
Wondering what props are in the `restProps`?

Reply:
Yeah, that's what I thought as well... that there wouldn't be anything in there. Happy to remove them